logs/events-2024-08-07_01-28-47.log

```diff
@@ -0,0 +1,4 @@
+2024-08-07 01:28:47,515 - Logging is enabled for preset: PRESET1.json
```

Comment:
lets not put the calendar date and time, but put the simulator time (time_elapsed) here instead.